### DISCUSSION TITLE
Welcome to solaredge Discussions!

### DISCUSSION THREAD

**Discussion by jbuehl** [jbuehl/solaredge#174] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eschwim** [comment]
Hello!  Just wanted to say "hi!" and a big thank you to @jbuehl for this project.

I started using it about 9 months ago after my first solaredge inverter died.   Because of the ... rather lackluster (read: non-existent) proactive alerting provided by the solaredge customer monitoring portal, my family didn't realize that our inverter had died until nearly 3 months after the fact.  Add another 6 weeks to get the RMA settled, and we missed out on pretty much the entire peak solar production season.

I fiddled around with trying to install a MITM proxy to intercept to TCP traffic, until I found out that the inverter updates they had pushed out made this impossible (they switched to native HTTPS; no idea why they feel compelled to encrypt our monitoring data other than to make their product as proprietary as possible).

After switching to using semonitor as an RS485 master (running on a raspberry PI powered by a PoE splitter and crammed in the inverter wiring + main power cutoff box) I was actually able to get all of the data that I needed!  I was happy to be able to contribute some code that allowed you to pipe this data to influxdb, which makes it super easy to set up a monitoring dashboard + alerts via Grafana.

And I'm happy/sad to say that I was recently able to reap the benefits of this work, because our inverter failed AGAIN last week.  Instead of noticing 3 months after the fact, we noticed the next day when we got an alert from our Grafana install!
